app/src/main/java/com/codepath/travelbud/parse_classes/Post.java

```diff
@@ -25,6 +32,17 @@ public class Post extends ParseObject {
     public static final String KEY_HASHTAGS = "hashtags";
     public static final String KEY_VISIBILITY = "visibility";
 
+//    @ColumnInfo(name = "description")
+//    private String description;
+//
+//    @ColumnInfo(name = "location_string")
+//    private String location_string;
+//
```

Comment:
Hey Kelly, I noticed these filed are no longer used, do we want to keep them in master? Maybe we could delete them?

Reply:
Yes, I'll delete them!